rows.go

```diff
@@ -115,24 +151,63 @@ func (rs *rowSets) invalidateRaw() {
 	rs.raw = nil
 }
 
+// Column is a mocked column Metadata for rows.ColumnTypes()
+type Column struct {
+	name, dbTyp              string
```

Comment:
can we make all these fields exported, removing `NewColumn` and `NewColumnSimple` functions. and move everything related to Column in `column.go` file

Reply:
@jopicornell these are the only changes I would like to be made here